pallets/rmrk-core/src/lib.rs

```diff
@@ -800,3 +688,140 @@ impl<T: Config> Collection<StringLimitOf<T>, T::AccountId> for Pallet<T> {
 		Ok(collection_id)
 	}
 }
+
+impl<T: Config> Nft<T::AccountId, StringLimitOf<T>> for Pallet<T> {
```

Comment:
How about moving trait impls to `functions` file?

Reply:
resolved in https://github.com/rmrk-team/rmrk-substrate/pull/28/commits/012dd72355ba4a8b52500f8b86f5056d0eff22db

Reply:
@bmacer changes are still visible in PR diff. need to update in line with `main`